C#/C#.sublime-syntax

```diff
@@ -1,278 +1,1634 @@
 %YAML 1.2
 ---
 # http://www.sublimetext.com/docs/3/syntax.html
-name: "C#"
+# https://github.com/gwenzek/csharpSyntax
```

Comment:
@FichteFoll That was a major oversight on our part, I'm sorry about that. We initially missed randy3k on the LaTeX one also. I'll see if I can put a list of major contributors and the syntaxes they've contributed in the readme.